test/Microsoft.AspNet.Mvc.TagHelpers.Test/OptionTagHelperTest.cs

```diff
@@ -401,7 +401,14 @@ public static IEnumerable<object[]> DoesNotUseViewContextDataSet
                 getChildContentAsync: () =>
                 {
                     var tagHelperContent = new DefaultTagHelperContent();
-                    tagHelperContent.SetContent(originalContent);
+                    if (string.IsNullOrEmpty(originalContent))
+                    {
+                        tagHelperContent.SetContent(originalContent);
+                    }
+                    else
+                    {
+                        tagHelperContent.SetContent("HtmlEncode[[" + originalContent + "]]");
+                    }
```

Comment:
why is this doing a fake encoding at this level?  what fails if `originalContent` is assigned without this?